web/modules/custom/ai_screening_project/src/Helper/ProjectHelper.php

```diff
@@ -461,4 +461,28 @@ public function loadProjectGroup(NodeInterface $project) : GroupInterface {
     return $this->groupStorage->load(reset($relationships)->getGroupId());
   }
 
+  /**
+   * Get a list of project track statuses and project status.
+   *
+   * @param string $projectId
+   *   The project id.
+   *
+   * @return array
+   *   A list of project status and track evaluation.
+   */
+  public function getProjectTrackEvaluation(string $projectId) : array {
+    $statuses = [];
+    $project = $this->nodeStorage->load($projectId);
+
+    if ($project instanceof NodeInterface) {
+      $projectTracks = $this->loadProjectTracks($project);
```

Comment:
If you add a `getProjectStats` function to this helper (and move code from the block helper into it), you can make do with 2 database queries to load all project tracks and make this more performant (by not loading tracks one at the time).